checkout/payment-menu.md

```diff
@@ -0,0 +1,419 @@
+|   | └─➔&nbsp;`preOrderPurchaseIndicator`  | `string`     | Indicates whether the payer is placing an order for merchandise with a future availability or release date. <br>`01` (Merchandise available) <br>`02` (Future availability)
+|   | └─➔&nbsp;`shipIndicator`              | `string`     | Indicates shipping method chosen for the transaction. <br>`01` (Ship to cardholder's billing address) <br>`02` (Ship to another verified address on file with merchant)<br>`03` (Ship to address that is different than cardholder's billing address)<br>`04` (Ship to Store / Pick-up at local store. Store address shall be populated in shipping address fields)<br>`05` (Digital goods, includes online services, electronic giftcards and redemption codes) <br>`06` (Travel and Event tickets, not shipped) <br>`07` (Other, e.g. gaming, digital service)
+|   | └─➔&nbsp;`giftCardPurchase`           | `bool`    | `true` if this is a purchase of a gift card.
+|   | └─➔&nbsp;`reOrderPurchaseIndicator`   | `string`     | Indicates whether the payer is placing an order for merchandise with a future availability or release date. <br>`01` (Merchandise available) <br>`02` (Future availability)
+|   | └➔&nbsp;`pickUpAddress`               | `object`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`name`                       | `string`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`streetAddress`              | `string`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`coAddress`                  | `string`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`city`                       | `string`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`zipCode`                    | `string`     | If `shipIndicator` set to `04`, then prefill this.
+|   | └─➔&nbsp;`countryCode`                | `string`     | If `shipIndicator` set to `04`, then prefill this.
+
+The response back should look something like this (abbreviated for brevity):
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+    "paymentorder": {
+      "id": "/psp/paymentorders/b80be381-b572-4f1e-9691-08d5dd095bc4"
+    },
+    "operations": [
+        {
+            "href": "https://ecom.externalintegration.payex.com/paymentmenu/core/scripts/client/px.paymentmenu.client.js?token=38540e86bd78e885fba2ef054ef9792512b1c9c5975cbd6fd450ef9aa15b1844&culture=sv-SE",
+            "rel": "view-paymentorder",
+            "method": "GET",
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| Property       | Type     | Description                                      |
+|:---------------|:---------|:-------------------------------------------------|
```

Comment:
```suggestion
| :------------- | :------- | :--------------------------------------------------------------------------------- |
```
This table as well does not format like the others in the project.

Reply:
It might be worth pointing out that I didn't actually change the formatting in any of the files and that is also not the point of this PR. If you are able to do it, please add commits to this branch so they are fixed as I'm unable to do it.